libraries/platform/iot_network.h

```diff
@@ -94,6 +111,178 @@ typedef void ( * IotNetworkReceiveCallback_t )( IotNetworkConnection_t pConnecti
                                                 void * pContext );
 /* @[declare_platform_network_receivecallback] */
 
+/**
+ * @brief Provide an asynchronous notification of network closing
+ *
+ * A function with this signature may be set with @ref platform_network_function_setclosecallback
+ * to be invoked when data is available on the network.
```

Comment:
This should read "to be invoked when network connection is closed"

Reply:
Fixed